### ISSUE TITLE
Turning off accessibility leaves markers on charts even with markers enabled false

### ISSUE THREAD

**Issue by TorsteinHonsi** [highcharts/highcharts#23783]
#### Expected behaviour
After disabling accessibility markers, existing marker graphics should be destroyed

#### Actual behaviour
They are only destroyed in some basic cases

#### Live demo with steps to reproduce
https://jsfiddle.net/highcharts/6rfezc4q/

#### Product version
12.4.0

#### Additional info
It's a narrower case of #16624. Destroying the markers in [the issue test](https://github.com/highcharts/highcharts/blob/aaba7e787d866db2732f6ddd25b0503ecc18a071/samples/unit-tests/accessibility/forced-markers/demo.js#L242-L275) works by coincidence because the data-sorting feature moves axis labels, which in turns marks the axis `isDirty`, which again marks the series `isDirty` and causes the re-rendering of the series and removal of graphics. If we set a different `xAxis.labels.step`, we cause the `moveLabel` logic not to run, and the series is never marked dirty.

I discovered this during refactoring of the data sorting feature, where the `moveLabel` function is removed.

The fix would be to make sure each Series' `isDirty` flag is set to true when relevant a11y features are set.

**Comment by parthz-13** [comment]
Hey @TorsteinHonsi , I think I should modify the function `chartUpdateA11yEnabled` in `ts/Accessibility/Accessibility.ts` and set each Series' `isDirty` flag to true (as you said). Should I go ahead with this idea and raise a PR?

**Comment by TorsteinHonsi** [comment]
@parthz-13 Yes that is probably safe, but we just need to make sure it's not too aggressive, forcing redraws when not necessary.

@oysteinmoseng @marvin19 any insight?

**Comment by parthz-13** [comment]
Hey @TorsteinHonsi, just wanted to check if you had the time to review the PR. Please let me know if there’s anything I should adjust or clarify.
Thanks!

**Comment by oysteinmoseng** [comment]
Setting `series.isDirty` makes sense, but it should be handled in `ForcedMarkers.ts`, not in `Accessibility.ts` to maintain proper separation of concerns. https://github.com/highcharts/highcharts/blob/master/ts/Accessibility/Components/SeriesComponent/ForcedMarkers.ts 

(This is only a problem due to the forced markers logic, which will be irrelevant with the new a11y module that is in progress)
